### PR TITLE
Added ability to infer CSV/TSV schema as well.

### PR DESCRIPTION
**Pull request by dkorolev**

### CODE REVIEW

File: Utils/JSONSchema/tsv2json.cc [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    return ExcelColName((i / 26) - 1) + static_cast<char>('A' + (i % 26));
+  }
+}
+
+int main() {
+  std::vector<std::map<std::string, std::string>> output;
+  std::string row_as_string;
+  while (std::getline(std::cin, row_as_string)) {
+    if (!row_as_string.empty()) {
+      const std::vector<std::string> fields = current::strings::Split(row_as_string, '\t', current::strings::EmptyFields::Keep);
```
Comment (mzhurovich) [inline Utils/JSONSchema/tsv2json.cc]: `make indent`?

File: Utils/JSONSchema/infer.h [hunk 2]
```diff
@@ -338,6 +338,10 @@ inline Schema RecursivelyInferSchema(const rapidjson::Value& value) {
     }
   } else if (value.IsString()) {
     return String(std::string(value.GetString(), value.GetStringLength()));
+#if 0  // Hack. -- D.K.
```
Comment (mzhurovich) [inline Utils/JSONSchema/infer.h]: Why is it needed?

Comment (dkorolev) [inline Utils/JSONSchema/infer.h]: Right now integers are not supported at all by this tool. I looked at the distribution of their values via this trick.

File: Utils/JSONSchema/tsv2json.cc (hunk 1 above)
Comment (dkorolev) [inline Utils/JSONSchema/tsv2json.cc]: :+1: 82c1d71

**Pull request merged**